sirepo/template/controls.py

```diff
@@ -328,8 +318,10 @@ def _add_ptc(instruments, data):
     v.initialCorrectors = '[{}]'.format(','.join([str(x) for x in c.corrector]))
     v.correctorCount = len(c.corrector)
     v.monitorCount = len(header)
+    v.schema = SCHEMA
     if i:
         _add_ptc(i, data.models.externalLattice)
+    v.isTrackingSim = True if LatticeUtil.find_first_command(data.models.externalLattice, 'ptc_create_universe') else False
```

Comment:
`bool(LatticeUtil.find_first_command(data.models.externalLattice, 'ptc_create_universe'))`. `If True else False` is almost always a code smell.